app/src/main/java/cz/applifting/humansis/repositories/ProjectsRepository.kt

```diff
@@ -16,16 +18,20 @@ import javax.inject.Singleton
 class ProjectsRepository @Inject constructor(val service: HumansisService, val dbProvider: DbProvider, val context: Context) {
 
 
-    suspend fun getProjectsOnline(): List<ProjectLocal>? {
+    suspend fun getProjectsOnline(sp: SharedPreferences): List<ProjectLocal>? {
```

Comment:
to jsme se vůbec nepochopili. Repository prostě nemá vůbec znát SharedPreferences. vždyť si můžeš poslat rovnou tu zemi, ne?